src/services/edusharing/logic/connector.js

```diff
@@ -18,6 +18,7 @@ class EduSharingConnector {
 		}
 		this.authorization = null; /* JSESSION COOKIE */
 		this.accessToken = null; /* ACCESSTOKEN */
+		this.accessTokenCreationTime = null; 
```

Comment:
Codacy found an issue: [Trailing spaces not allowed.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=6005161)